### PR TITLE
Use ssh config even if private key not present

### PR DESCRIPTION
**Pull request by kvaidas**

### CODE REVIEW

File: terraform/terraform-0.12.sh [hunk 1]
```diff
@@ -48,12 +48,12 @@ setup_ssh() {
...
-        mkdir -p ~/.ssh
-        cat > ~/.ssh/config <<EOF
+    mkdir -p ~/.ssh
+    cat > ~/.ssh/config <<EOF
 StrictHostKeyChecking no
 LogLevel quiet
 EOF
-        chmod 0600 ~/.ssh/config
+    chmod 0600 ~/.ssh/config
   fi
```
Comment (itsdalmo) [inline terraform/terraform-0.12.sh]: The `fi` has to enclose the if-statement still 👍

**Comment by itsdalmo** [comment]
Looks great 💯 Could you give it a spin in an actual pipeline with/without a `github_private_key` and just verify that it works?

**Review by itsdalmo — APPROVED** [review]
Tested this now and it works great. Thanks 💯

**Comment by kvaidas** [comment]
I didn't have a testcase ready so thank you for testing it! :)

**Pull request merged**